### DISCUSSION TITLE
Dashboard Loads But The Previous Tests/Projects Are Missing, After trying to create new project application stops responding.

### DISCUSSION THREAD

**Discussion by atillaersoy** [ludeknovy/jtl-reporter#249] [Q&A]
Hello there,

Dashboard Loads But The Previous Tests/Projects Are Missing, After trying to create new project application stops responding.

I'm providing -be logs below.

Any idea? 

0 verbose cli /usr/local/bin/node /usr/local/bin/npm
1 info using npm@8.19.3
2 info using node@v18.13.0
3 timing npm:load:whichnode Completed in 1ms
4 timing config:load:defaults Completed in 9ms
5 timing config:load:file:/usr/local/lib/node_modules/npm/npmrc Completed in 2ms
6 timing config:load:builtin Completed in 2ms
7 timing config:load:cli Completed in 14ms
8 timing config:load:env Completed in 1ms
9 timing config:load:file:/src/be/.npmrc Completed in 0ms
10 timing config:load:project Completed in 7ms
11 timing config:load:file:/root/.npmrc Completed in 0ms
12 timing config:load:user Completed in 2ms
13 timing config:load:file:/usr/local/etc/npmrc Completed in 1ms
14 timing config:load:global Completed in 1ms
15 timing config:load:validate Completed in 3ms
16 timing config:load:credentials Completed in 9ms
17 timing config:load:setEnvs Completed in 2ms
18 timing config:load Completed in 54ms
19 timing npm:load:configload Completed in 56ms
20 timing npm:load:mkdirpcache Completed in 31ms
21 timing npm:load:mkdirplogs Completed in 1ms
22 verbose title npm run start
23 verbose argv "run" "start"
24 timing npm:load:setTitle Completed in 4ms
25 timing config:load:flatten Completed in 31ms
26 timing npm:load:display Completed in 34ms
27 verbose logfile logs-max:10 dir:/root/.npm/_logs
28 verbose logfile /root/.npm/_logs/2023-11-08T13_17_29_475Z-debug-0.log
29 timing npm:load:logFile Completed in 37ms
30 timing npm:load:timers Completed in 2ms
31 timing npm:load:configScope Completed in 0ms
32 timing npm:load Completed in 169ms
33 silly logfile start cleaning logs, removing 1 files
34 silly logfile done cleaning log files
35 timing command:run Completed in 99284ms
36 verbose stack Error: command failed
36 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/lib/index.js:63:27)
36 verbose stack     at ChildProcess.emit (node:events:513:28)
36 verbose stack     at maybeClose (node:internal/child_process:1091:16)
36 verbose stack     at ChildProcess._handle.onexit (node:internal/child_process:302:5)
37 verbose pkgid jtl-reporter-be@1.0.0
38 verbose cwd /src/be
39 verbose Linux 5.15.90.1-microsoft-standard-WSL2
40 verbose node v18.13.0
41 verbose npm  v8.19.3
42 error path /src/be
43 error command failed
44 error signal SIGTERM
45 error command sh -c -- node ./dist/server.js
46 verbose exit 1
47 timing npm Completed in 99509ms
48 verbose code 1
49 error A complete log of this run can be found in:
49 error     /root/.npm/_logs/2023-11-08T13_17_29_475Z-debug-0.log

**Comment by ludeknovy** [comment]
Hi @atillaersoy how do you run the app? Do I see correctly that you run it in WSL and not in the docker container?

**Reply by atillaersoy** [reply]
I actually do run it on windows machine installed on docker

**Reply by ludeknovy** [reply]
I can help a little with windows, unfortunately. Did you try to use the WSL 2 based engine in docker settings?

**Reply by atillaersoy** [reply]
Any kind of help is appreciated :) Yes, it is WSL 2 based engine. It was fairly stable since months ago but now can't access it

**Reply by ludeknovy** [reply]
Honestly, I cannot read the root cause from the provided logs. Like the backend service won't start properly, but that's all I can read from there.

**Reply by atillaersoy** [reply]
Is there any other logs you may want? So we can debug this further. Actually i'm just curious if we can restore the data to fresh installation in order to prevent any data loss.

Btw thanks for the support

**Reply by ludeknovy** [reply]
If those are the logs from the be docker container I don't know what more to ask for :|

You can back up the DB and restore it after you make it work again:
https://jtlreporter.site/docs/guides/backup-and-restore-database
just be sure you use the same version as before to avoid any schema collisions.
